contexts/reducers/change-balance.ts

```diff
@@ -15,7 +15,8 @@ export const ChangeBalance: ReduceAction<Partial<BalanceState> | BalanceState> =
     fn: reducer
   };
 
-export const changeBalance = (payload: Partial<BalanceState> | BalanceState): ReduceActor<Partial<BalanceState> | BalanceState> => ({
+export const changeBalance = 
+(payload: Partial<BalanceState> | BalanceState): ReduceActor<Partial<BalanceState> | BalanceState> => ({
```

Comment:
missing space after `=\n` add 2 spaces to this line so it's correctly indented 

you can also add type for the junctions used to reduce character count,

```ts
type PayloadParam = Partial<BalanceState> | BalanceState;

export const changeBalance = (payload: PayloadParam): ReduceActor<PayloadParam> => // ... rest of code
```